user_guide/intro_log_in_and_edit_profile.rst

```diff
@@ -0,0 +1,554 @@
+Most of the actions is described in the :ref:`Productivity Tools section <user-guide-productivity-tools>`.
+
+.. image:: ./img/intro/user_view_actions.png 
+
+
+Add and Edit Tags
+"""""""""""""""""
+
+If you a permitted, you can add and edit tags for your user. The tags can be used for the search, reporting, 
+segmentation and filtering.
+
+- Hover the mouse over the *"Tags"* field in the *"General"* section of the *"My User"* page as shown below.
+
+  |
+
+  |IntroTags|
+
+  |
+  
+- Click the |IcEdit| icon, and the tag field will appear.
+
+  |
+  
+  |IntroTags1|
+
+  |
+  
+- Start entering a tag and if an existing tag is found, it will be displayed for selection. If not, a new tag will be 
+  created.  
+
+- If you have entered a new tag, it will appear in the selector with a *"(New tag)"* label. Select this to add to your 
+  user as well as the list of available tags for future use. This way you can add one or more tags that contain one or 
+  several words.
+  
+  |
+  
+  |IntroTags2|
+
+  |
+  
+Generate an API Key
+"""""""""""""""""""
+
+Should an integration with a third-party software require an 
+`API Key <https://en.wikipedia.org/wiki/Application_programming_interface_key>`_, it can be generated  on the "My User" 
+page:
+
+|
+
+.. image:: ./img/intro/user_view_apik.png
+
+|
+
+- Click the :guilabel:`Generate Key` button next to the API Key label 
+
+- Copy and use the key generated
+
+.. caution::
+
+    Only one API key may exist for the user at the same time. If you choose to generate another API key over the 
+    existing one, the old key is discarded and cannot be used to access OroCRM.
+    
+    
+Download the Outlook Add-in
+"""""""""""""""""""""""""""
+
+OroCRM Enterprise Edition users will also find a link to the latest version of OroCRM add-in for Outlook. 
+With this add-in you can synchronize contacts, tasks, and calendar events between OroCRM and your Outlook.  
+You can associate emails to accounts, contacts, opportunities, and cases and create leads, opportunities and cases 
+in OroCRM based on emails from the Outlook side bar.
+
+|
+
+.. image:: ./img/intro/user_outlook.png
+
+|
+
+Click the link to start the download. The file will be saved subject to your browser settings. 
+
+To learn more about the add-in, please refer to the :ref:`Integration with Outlook guide <user-guide-synch-outlook>`.
+
+
+Get to the Edit Page of Your User Profile
+"""""""""""""""""""""""""""""""""""""""""    
+Click the :guilabel:`Edit` button in the top left corner of the page to change and add the profile details, as described 
+in the section below.
+
+|
+
+.. image:: ./img/intro/user_edit.png
+    
+|
+    
+.. _user-guide-getting-started-profile:
+    
+Edit Your User Profile
+-----------------------
+
+Once you have clicked the Edit, the Edit form of your user profile will be displayed. Here you can update your 
+credentials, change your password, and synchronize OroCRM with an existing IMAP mailbox.
```

Comment:
Once you have clicked Edit, the Edit form of your user profile will be displayed. Here you can update your credentials, change your password, and synchronize OroCRM with an existing IMAP mailbox.